rasa/nlu/extractors/extractor.py

```diff
@@ -78,24 +86,6 @@ def add_processor_name(self, entity: Dict[Text, Any]) -> Dict[Text, Any]:
 
         return entity
 
-    def init_split_entities(self) -> Dict[Text, bool]:
```

Comment:
This function is pointless, we can just call `rasa.utils.train_utils.init_split_entities` directly, like in `TEDPolicy`.